### PR TITLE
Add pylint testing

### PR DESCRIPTION
**Pull request by SalatskySal**
Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Pull request includes link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 1]
```diff
@@ -393,8 +393,7 @@ def _write_demultiplexed_logs(self, build_id, logs_dir):
                 platform_logs[platform].write((entry.line + '\n').encode('utf-8'))
                 platform_logs[platform].flush()
             except Exception as error:
-                msg = "Exception (%s) while writing build logs: %s" % (type(error),
-                                                                       error)
+                msg = "Exception (%s) while writing build logs: %s", type(error), error
                 raise ContainerError(msg)
```
[the remark this answers is not in the document]
Comment (SalatskySal) [reply koji_containerbuild/plugins/builder_containerbuild.py:400]: > Looks good, just one issue :)
> 
> Last commit is also missing "Signed-off-by" and should probably be part of "Fix single errors" anyway

Oh, I forgot to rebase this :)

File: tests/test_builder_containerbuild.py [hunk 2]
```diff
@@ -29,11 +29,10 @@
 import osbs
 import pytest
 from flexmock import flexmock
-from osbs.exceptions import OsbsValidationException
 try:
     from osbs.exceptions import OsbsOrchestratorNotEnabled
 except ImportError:
-    from osbs.exceptions import OsbsValidationException as OsbsOrchestratorNotEnabled
+    import osbs.exceptions.OsbsValidationException as OsbsOrchestratorNotEnabled
```
[the remark this answers is not in the document]
Comment (SalatskySal) [reply tests/test_builder_containerbuild.py:36]: I don't know, why I changed that, but if I remember it right, it's related to re-import

[the remark this answers is not in the document]
Comment (chmeliik) [reply tests/test_builder_containerbuild.py:36]: I see, I'd probably just mute this one, it's not worth fighting pylint over an intended workaround :D

[the remark this answers is not in the document]
Comment (SalatskySal) [reply tests/test_builder_containerbuild.py:36]: Hmm, you're right, it was the unneeded change, it's not related to origin pylint warning

File: koji_containerbuild/plugins/builder_containerbuild.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (SalatskySal) [reply koji_containerbuild/plugins/builder_containerbuild.py]: Fixed

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 3]
```diff
@@ -393,8 +393,7 @@ def _write_demultiplexed_logs(self, build_id, logs_dir):
                 platform_logs[platform].write((entry.line + '\n').encode('utf-8'))
                 platform_logs[platform].flush()
             except Exception as error:
-                msg = "Exception (%s) while writing build logs: %s" % (type(error),
-                                                                       error)
+                msg = "Exception (%s) while writing build logs: %s", type(error), error
```
Comment (MartinBasti) [inline koji_containerbuild/plugins/builder_containerbuild.py]: Explain this please

Comment (MartinBasti) [reply koji_containerbuild/plugins/builder_containerbuild.py]: Why do you pass tuple to exception?

Comment (SalatskySal) [reply koji_containerbuild/plugins/builder_containerbuild.py]: This was a mistake from my side and this has already fixed

**Comment by MartinBasti** [comment]
no release notes internal change only

**Pull request merged**